releasenotes/notes/47655.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+- |
+  **Added** `ENABLE_EXTERNAL_NAME_SERVICE` feature flag, which allows istio to skip processing ExternalName service, it can be used to get rid of ExternalName service's influence completely.
```

Some minor changes, mostly to mention the default is true, and one may set it to false.
```suggestion
  **Added** `ENABLE_EXTERNAL_NAME_SERVICE` feature flag, which defaults to true conserving the current behavior processing the ExternalName service. It can be set to false removing the ExternalName service's influence completely.
```